engine/BUILD.bazel

```diff
@@ -0,0 +1,66 @@
+load("//:version.bzl", "PROJECT_VERSION")
+
+cc_library(
+    name = "enginelib",
+    hdrs = glob(["src/**/*.hpp"], exclude=["src/main*.hpp"]),
+    srcs = glob(
+        ["src/**/*.cpp"],
+        exclude=[
+            "src/main*.cpp",
+            "src/**/*_test.cpp",
+            "src/server_mock.cpp",
+        ]
+    ),
+    data = glob(["lua/**"]),
+    includes = ["src"],
+    additional_compiler_inputs = glob(["webui/**"]),
+    deps = [
+        "//engine/vendor/lrdb",
+        "//engine/vendor/linenoise",
+        "//stack",
+        "//fable",
+        "//runtime",
+        "//models",
+        "//oak",
+        "@boost//:algorithm",
+        "@boost//:conversion",
+        "@boost//:filesystem",
```

Comment:
nitpick, hopefully you can drop this dependency eventually when you drop < gcc-9.2 support

Reply:
Can already be removed, actually.